.github/workflows/enforce-pr-labels-release.yml

```diff
@@ -0,0 +1,35 @@
+name: Enforce Release PR labeling
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, edited, synchronize]
```

Comment:
Do you happen to know what a "synchronized" pull request is?  See any other ones we would want to add from the docs below?

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Reply:
I believe it means that the PR is from a fork and the fork was updated. I went through those docs and this is the full list I think we need.

Reply:
oooh, other than `reopened`, which I added to some but apparently not all. Will make sure all have it.